refarch-frontend/src/composables/formatter.ts

```diff
@@ -0,0 +1,15 @@
+import moment from "moment";
+
+export function useFormatter() {
+  function startingCharUpperCase(text: string): string {
+    return text
+      ? text.toLowerCase().replace(/^\w/, (c) => c.toUpperCase())
+      : "";
+  }
+
+  function formatDate(date: Date): string {
+    return date ? moment(date).format("L") : "";
+  }
+
+  return { startingCharUpperCase, formatDate };
+}
```

Comment:
This file tries to mimic a Vue composable to expose the mentioned methods. However no Vue specific stuff like Refs etc. is used here so this can be used totally outside the Vue "ecosystem". That's why I suggest to refactor and rename this file (e.g. a static class) to indicate its not Vue dependent.